plugins/evernote.koplugin/main.lua

```diff
@@ -405,6 +405,7 @@ function EvernoteExporter:doLogin(username, password)
         domain = self.evernote_domain,
         username = username,
         password = password,
+        logger = DEBUG
```

Comment:
Would you mind changing them all to logger.dbg? That `DEBUG` is kind of a blast from the past.

More important, we prefer a trailing comma at the end. ;-)

Reply:
Sure